stdlib/builtins.pyi

```diff
@@ -91,6 +91,9 @@ _SupportsAnextT = TypeVar("_SupportsAnextT", bound=SupportsAnext[Any], covariant
 _AwaitableT = TypeVar("_AwaitableT", bound=Awaitable[Any])
 _AwaitableT_co = TypeVar("_AwaitableT_co", bound=Awaitable[Any], covariant=True)
 _P = ParamSpec("_P")
+_StartT = TypeVar("_StartT", default=int)
+_StopT = TypeVar("_StopT", default=_StartT)
+_StepT = TypeVar("_StepT", default=int | None)
```

Comment:
Maybe as a very first step, try using `Any` as default. We could change this later, but this should allow explicit annotations without regressions.